IntegrationTests/tests_04_performance/test_01_resources/shared.swift

```diff
@@ -136,3 +136,119 @@ func withAutoReleasePool<T>(_ execute: () throws -> T) rethrows -> T {
     }
     #endif
 }
+
+// MARK: UDP Shared
+
+/// Repeatedly send, echo and receive a number of UDP requests.
+/// Based on the UDPEchoClient/Server example.
+enum UdpShared {
+    private final class EchoHandler: ChannelInboundHandler {
+        public typealias InboundIn = ByteBuffer
+        public typealias OutboundOut = ByteBuffer
+
+        public func channelRead(context: ChannelHandlerContext, data: NIOAny) {
+            // As we are not really interested getting notified on success or failure we just pass nil as promise to
+            // reduce allocations.
+            context.write(data, promise: nil)
+        }
+
+        public func channelReadComplete(context: ChannelHandlerContext) {
+            // As we are not really interested getting notified on success or failure we just pass nil as promise to
+            // reduce allocations.
+            context.flush()
+        }
+
+        public func errorCaught(context: ChannelHandlerContext, error: Error) {
+            print("error: ", error)
```

Comment:
do we really expect errors? If not, we should just crash on error. `print` would also allocate loads

Reply:
Nope.  Will change.

Reply:
now fatal.